tools/resource_analysis.py

```diff
@@ -1,18 +1,27 @@
+# This file does some basic analysis on the intersection of the resources
+# with the highest usage
+
 import pickle
 from resource_map_creator import Resource
 
 resource_map = pickle.load( open("../../data/resource_map.pkl", "rb") )
 
 most_used = set()
 
-for i in range(4):
```

Comment:
I believe this is a double nested for loop however, it's a little ambigous. Maybe add an indentation to follow style guide.

Reply:
I'm a little confused. I removed this line. The line below it IS the outer part of a nested for loop. I think its easier to just do something that is O(10n) than to sort and grab the top ten elements.